baybe/surrogates/gaussian_process/presets/core.py

```diff
@@ -0,0 +1,26 @@
+"""Preset configurations for Gaussian process surrogates."""
+
+from __future__ import annotations
+
+from enum import Enum
+from typing import TYPE_CHECKING
+
+if TYPE_CHECKING:
+    from baybe.surrogates.gaussian_process.core import GaussianProcessSurrogate
+
+
+class GaussianProcessPreset(Enum):
+    """Available Gaussian process surrogate presets."""
+
+    DEFAULT = "DEFAULT"
```

Comment:
bit surprised you didnt already inlcude the versions `EDBO`, `BoTorch` here? Subsequent PR?
Perhaps we should also discuss what sets we think would be available, here my thoughts:
- `EDBO` adjusted for rdkit but otherwise exact copy of edbo
- `BoTorch` using the botorchd efaults
- `BayBE/Default`, smooth transition from lowD to highD priors taken from EDBO

Reply:
Had the exact same thoughts. The reasons why I thought it's better to delay it to a subsequent PR:
* Technically new features (in the previous code, there was effectively only the `default`)
* Getting the botorch defaults for transfer learning requires a bit of diggin, which might become obsolete once #130  is merged because we can then switch to the native botorch implementation. Hence, would prefer to wait at least with the `BOTORCH` preset a bit, to avoid unnecessary work
* For our own preset, I think we really need to set up the a benchmark first in order to figure out how to even configure it.

So the only "easy" preset that could be added right now is "EDBO". Shall I do so? 

And BTW: do you prefer `DEFAULT` or `BAYBE` as preset name?

Reply:
ok lets forget about the botorch preset in here
perhaps `BayBE` is a better name because then on can say `the default is BayBE`

Reply:
Ok, renamed to BAYBE. 
And have decided to to the EDBO preset in an immediate follow-up PR because it requires the likelihood extension, which I'll do immediately together with the mean extension and then this PR will just be too large. So let me close here